### DISCUSSION TITLE
CHAMILO 1.11.26 : Import SCORM ne marche pas

### DISCUSSION THREAD

**Discussion by mahmoudmadiou** [chamilo/chamilo-lms#5769] [General]
Bonjour
Je fais face à un problème d’import de fichier SCORM dans un parcours.
J’ai 2 hébergements identiques chez A2Hosting
J’ai installé CHAMILO via leur service Softaculous (one click install)
Sur la première instance - Chamilo 1.11.20, l’import d’un fichier SCORM dans un parcours fonctionne sans problème
Sur l’autre instance - Chamilo 1.11.26 (installée en septembre 2023) , l’import du même fichier et d’autres aussi fonctionne pas, ça reste bloqué à 0%
 
Certes, J’ai trouvé un contournement très fastidieux, où je crée dans l’instance 1 et importe dans l'instance 2, puis charger les fichiers zip manuellement, etc …

Une idée de l'origine de cette anomalie ?

Merci

**Comment by NicoDucou** [comment]
Bonjour @mahmoudmadiou 

Il y a un problème dans la 1.11.26 avec l'import de fichier avec des noms contenant des caractères spéciaux.
Il faut renommer le fichier Scorm mais également les fichiers images qui sont inclus ne doivent pas avoir de caractère spécial.
C'est corrigé pour la prochaine version 1.11.28 mais en attendant il faut faire attention au nom de fichier ainsi qu'au nom des fichiers qui sont inclus dans le scorm.

Cordialement

**Reply by mahmoudmadiou** [reply]
Bonjour, merci pour ces informations. 

Cordialement
